src/proto/ast.proto

```diff
@@ -202,6 +237,9 @@ message Statement {
 	optional ChangeKind structural_change_kind = 10;
 	/** Kind of change on the label: UNCHANGED, ADDED, DELETED/REMOVED, RENAMED */
 	optional ChangeKind label_change_kind = 11;
+	/** For Specs use */
+	optional SpecStatement specs = 12;
```

Comment:
same thing - 'repeated' or optional?

Reply:
based on the naming of 'specs' (plural) I think we intended repeated here.

Reply:
Done.